model/Core/Properties/devLifeCycleStage.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# devLifeCycleStage
```

Comment:
I am not a native English speaker but I think we can simply do `devLifecycleStage` because lifecycle is really one word.